conftest.py

```diff
@@ -121,7 +121,7 @@ def activite(db):
         "Aide à la personne",
         "Aide sociale à l'enfance : action éducative",
         "Ambulances",
-        "Aménagement maison : cuisine salle de bain salon",
```

Comment:
Is it a breaking space vs non breaking space change ?

Reply:
Yes, your best friend.